Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,22 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+function printReceipt(order) {
+  console.log("QTY\tITEM\t\t\tTOTAL");
+
+  let totalPrice = 0;
+
+  order.forEach(({ itemName, quantity, unitPricePence }) => {
+    let totalItemPrice = (unitPricePence * quantity) / 100; // Convert pence to pounds
+    totalPrice += totalItemPrice;
+
+    // Format output for better alignment
+    console.log(`${quantity}\t${itemName.padEnd(25)} £${totalItemPrice.toFixed(2)}`);
+  });
+
+  console.log("\nTotal: £" + totalPrice.toFixed(2));
```

Comment:
Can you align the TOTAL column header with the amount. The currency symbol is lined up underneath the 'T'.

![Screenshot 2025-04-20 at 8 12 15 PM](https://github.com/user-attachments/assets/440449b6-0d0b-4a23-a078-042a67f90950)